shared/unattended/RHEL-8-devel.ks

```diff
@@ -0,0 +1,99 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+poweroff
+lang en_US.UTF-8
+keyboard us
+network --onboot yes --device eth0 --bootproto dhcp
+rootpw --plaintext 123456
+firstboot --disable
+user --name=test --password=123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+KVM_TEST_LOGGING
+clearpart --all --initlabel
+autopart
+xconfig --startxonboot
+# Additional repositories could be specified in 'kickstart_extra_repos' parameter
+KVM_TEST_REPOS
+
+%packages --ignoremissing
+@base
+@core
+@development
+@additional-devel
+@debugging
+@network-tools
+@gnome-desktop
+@fonts
+@smart-card
+python3-pillow
+python3-six
+python3-pyparsing
```

Comment:
I have copied almost all from `shared/unattended/RHEL-7-devel.ks` file. Some packages here are not provided by RHEL8 any more. So I have replaced them with their `python3` equivalent.
`python-imaging` -> `python3-pillow`
`python-six` -> `python3-six`
`pyparsing` - > `python3-pyparsing`
there are some obsoletes: `ntpdate`, `spice-xpi`, so I removed both.

Reply:
IIRC ntpdate is used by some tests, do you know what replaced it and whether it's possible to get ntpdate on RHEL8?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/configuring_basic_system_settings/#migrating-to-chrony_using-chrony-to-configure-ntp

I guess we can replace it with chronyd.
```bash
# ntpdate ntp.example.com
# chronyd -q 'server ntp.example.com iburst'
```

Reply:
Yes, I knew about it. I tried this ks file and `chrony` package is installed by default so IMHO no need to add anything.

Reply:
Thanks for the clarification, let's hope someone fixes the tests...